docs/table_design/Data_distribution.md

```diff
@@ -59,38 +59,40 @@ For example, if you create a table without specifying the data distribution meth
 
 ```SQL
 CREATE TABLE site_access(
-    site_id INT DEFAULT '10',
-    city_code SMALLINT,
-    user_name VARCHAR(32) DEFAULT '',
-    pv BIGINT SUM DEFAULT '0'
+    event_day DATE,
+    site_id INT DEFAULT '10', 
+    pv BIGINT DEFAULT '0' ,
+    city_code VARCHAR(100),
+    user_name VARCHAR(32) DEFAULT ''
 )
-AGGREGATE KEY(site_id, city_code, user_name); -- The data distribution method is not specified.
+DUPLICATE KEY(event_day,site_id,pv); -- The data distribution method is not specified.
 ```
 
 Specify Hash distribution as the data distribution method at table creation.
 
 ```SQL
 CREATE TABLE site_access(
+    event_day DATE,
     site_id INT DEFAULT '10',
     city_code SMALLINT,
     user_name VARCHAR(32) DEFAULT '',
     pv BIGINT SUM DEFAULT '0'
 )
-AGGREGATE KEY(site_id, city_code, user_name)
-DISTRIBUTED BY HASH(site_id); -- Set the bucketing method as Hash bucketing and specify the bucketing key.
+AGGREGATE KEY(event_day, site_id, city_code, user_name)
+DISTRIBUTED BY HASH(event_day,site_id); -- Set the bucketing method as Hash bucketing and specify the bucketing key.
 ```
 
 Specify Range + Random distribution as the data distribution method at table creation.
 
 ```SQL
 CREATE TABLE site_access(
     event_day DATE,
-    site_id INT DEFAULT '10',
+    site_id INT DEFAULT '10', 
+    pv BIGINT DEFAULT '0' ,
     city_code VARCHAR(100),
-    user_name VARCHAR(32) DEFAULT '',
-    pv BIGINT SUM DEFAULT '0'
+    user_name VARCHAR(32) DEFAULT ''
 )
-AGGREGATE KEY(event_day, site_id, city_code, user_name)
+DUPLICATE KEY(event_day,site_id,pv)
 PARTITION BY RANGE(event_day) ( -- Set the partitioning method as Range partitioning.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/5d8e46ff-ea26-4e8d-941c-a3332ffdc28d)
suggest moving this left parenthesis down to the next line. (See to the next code demo too).
Also, I suggest using different table names in different code demos could be better, because users may create all these tables, but the database does not allow tables with the same name and may cause table creation failures. But this is a small issue. It's your call.